src/librustc/mir/repr.rs

```diff
@@ -162,6 +163,24 @@ impl<'tcx> IndexMut<BasicBlock> for Mir<'tcx> {
     }
 }
 
+impl From<usize> for BasicBlock {
```

Comment:
Maybe we should rewrite the graph trait to use the `Idx` trait from `IndexVec` -- this might entail moving `IndexVec` to `rustc_data_structures` (if it's not there already, I can't remember -- but it's probably where it belongs).